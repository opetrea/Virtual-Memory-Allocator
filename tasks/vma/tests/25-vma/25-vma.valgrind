==91923== Memcheck, a memory error detector
==91923== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==91923== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==91923== Command: /home/student/Desktop/tema1_sda/sd-2023-homeworks/vma
==91923== Parent PID: 91922
==91923== 
==91923== 
==91923== HEAP SUMMARY:
==91923==     in use at exit: 0 bytes in 0 blocks
==91923==   total heap usage: 13 allocs, 13 frees, 8,428 bytes allocated
==91923== 
==91923== All heap blocks were freed -- no leaks are possible
==91923== 
==91923== For lists of detected and suppressed errors, rerun with: -s
==91923== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
