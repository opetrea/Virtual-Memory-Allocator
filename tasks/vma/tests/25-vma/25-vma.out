Warning: size was bigger than the block size. Reading 5 characters.
GIGEL
